source/posts/2016-06-21-dont-let-sunk-costs-sink-your-project.html.markdown.erb

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "Don't let sunk costs sink your project"
+type: post
+author: Jeff Norris
+excerpt: "When we develop software projects, we can get into situations where we are also high-centered, places where it is difficult to make progress either forwards or backwards."
+title_tag_of_header: "Don't let sunk costs sink your project | GoCD Blog"
+meta_description: "When we develop software projects, we can get into situations where we are also high-centered, places where it is difficult to make progress either forwards or backwards."
+meta_keywords: "GoCD, continuous delivery, goforcd, open source, software, contribute to open source, agile continuous delivery, code contribution, videos, Go, performance testing, pipeline"
+summary_image: "/assets/images/blog/high-centered-team/sunkcosts_masthead.jpg"
+---
+
+When my family and I moved out of our last house, we decided to do the majority of the hauling ourselves using a rental truck. Our house was up on the side of a hill with an uncomfortably steep driveway. We picked up the rental truck early in the day, pulled it up to the house and spent the majority of the day filling it up. When we finally finished loading it, we were exhausted, hungry, and really anxious to get across town to start unloading. I carefully backed the truck out of our driveway while my wife made sure I avoided the tree branches and the mailbox. Everything was fine until I got to that spot where the driveway met the street, where the truck just… stopped moving.
+
+<figure>
+  <img src="/assets/images/blog/high-centered-team/truck.jpg"
```

Comment:
I should probably explain why we do this :) If we were to migrate the blog or the directory containing images over to a different location, it's much easier to do it if you're linking the image programatically than by specifying an absolute location.